src/platform/forms/tests/components/common/grid/Column.unit.spec.jsx

```diff
@@ -0,0 +1,42 @@
+// libs
+import React from 'react';
+import { render } from '@testing-library/react';
+import { expect } from 'chai';
+
+import Column from 'platform/forms/components/common/grid/Column';
+
+describe('Common Column component', () => {
+  it('should render', () => {
+    const tree = render(<Column testId="12345" />);
+
+    expect(tree.getByTestId('12345')).to.not.be.null;
+    tree.unmount();
+  });
+
+  it('should render child components', () => {
+    const childComponent = <div data-testid="12345">i am a component</div>;
+
+    const tree = render(<Column>{childComponent}</Column>);
+
+    expect(tree.getByTestId('12345')).to.not.be.null;
+    tree.unmount();
+  });

```

Comment:
Like I mentioned earlier, what do you think of this pattern that avoids relying on `data-testid`?

```js
  it('should render child components', () => {
    const childComponent = <div>i am a component</div>;

    const tree = render(<Column>{childComponent}</Column>);

    expect(tree.getByText('i am a component')).to.not.be.null;
    tree.unmount();
  });
```

Reply:
We have options:
https://testing-library.com/docs/dom-testing-library/api-queries

I've always preferred the data- approach to querying the dom for testing.